drinks/tasks/models.py

```diff
@@ -1,3 +1,28 @@
 from django.db import models
+from django.contrib.auth.models import User
+from django.core.validators import MinValueValidator, MaxValueValidator
+
+
+class Ingredient(models.Model):
+    unit_type = [('ml', 'mililitry'), ('szt', 'sztuk')]
+    name = models.CharField(max_length=220)
+    ingredient_amount = models.PositiveIntegerField(default=1,
+                                                    validators=[MinValueValidator(1),
+                                                                MaxValueValidator(300)],
+                                                    blank=True)
+    unit = models.CharField(default='ml', choices=unit_type, max_length=3)
+
+    def __str__(self):
+        return f'{self.name}  |  {self.ingredient_amount} {self.unit}'
+
+
+class Drink(models.Model):
+    drink_name = models.CharField(max_length=100, blank=False)
```

Comment:
tak samo jak w komentarzu wyżej, nie trzeba dodawać, że to jest drink name skoro nazwa klasy na to wskazuje

Reply:
uproszczono nazwę do name